### ISSUE TITLE
Fix wrong HTTP status code for GET /users/<id> API

### ISSUE THREAD

**Issue by isabelcosta** [anitab-org/mentorship-backend#343]
## Description
As a frontend developer,
I need to get correct status code,
so that I can code the frontend accordingly.

The `GET /users/<user_id>` API has wrong HTTP Status code being returned when the request is successful. The correct code should be 200 instead of 201, since no new resource is created.
 
### Update [Required]
- [ ] Fix wrong HTTP status code
- [ ] Update tests that may break with this change

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
1 hours

**Comment by jotharman** [comment]
@isabelcosta I am a first-timer. Can I take up this issue?

**Comment by StergiosSozos** [comment]
Hello!
This is my first time contributing to an open source project. Can I work on this issue?

**Comment by isabelcosta** [comment]
i think you both were assigned issues already right? @jotharman @StergiosSozos :) Let me know if this is not the case please

**Comment by jotharman** [comment]
yes, I have been assigned issue #475  and I have made required changes for that and I am looking for another one :)

**Comment by bera5186** [comment]
@isabelcosta if this issue is not assigned to anyone please assign this to me

**Comment by isabelcosta** [comment]
@bera5186 you were assigned an issue already (one you asked for), so I won't assign you this one.

**Comment by yashikabaan** [comment]
@isabelcosta 
Is this issue still open?
I am a first timer and would like to work on this issue

**Comment by vibhuti019** [comment]
@isabelcosta  I wish to work on this issue.

**Comment by isabelcosta** [comment]
@vibhuti019 before assigning to you I would like to ask @yashikabaan 
@yashikabaan sorry for the delay in my response, are you still interested? I can assign it to you still :)

**Comment by Shivansh2407** [comment]
@isabelcosta
if( This Issue == Fixed):
    Please Close the Issue
else:
    I would like to fix this

**Comment by isabelcosta** [comment]
@Shivansh2407 
If (no previous responses):
    @Shivansh2407 feel free to work on this
    return issue assigned

**Comment by vj-codes** [comment]
Unassigning @Shivansh2407 due to inactivity.

**Comment by yash2189** [comment]
Can I try working on this @isabelcosta @vj-codes ?

**Comment by Shivansh2407** [comment]
Yes @yash2189, go ahead 👍

**Comment by yash2189** [comment]
Hello I created a new user and tested the `GET /users/<user_id>`API It is showing 200 
(https://github.com/yash2189/mentorship-backend/blob/develop/snap_1.PNG)

**Comment by yash2189** [comment]
@Shivansh2407 can you please have a look at this? 

> Hello I created a new user and tested the `GET /users/<user_id>`API It is showing 200
> (https://github.com/yash2189/mentorship-backend/blob/develop/snap_1.PNG)

**Comment by yash2189** [comment]
Unassigning from this please make it available for others.

**Comment by justdvnsh** [comment]
@vj-codes I'd like to take this issue up. Can I be assigned to this issue ?

**Comment by vj-codes** [comment]
@justdvnsh are you contributing for the first time to this repo? If yes ,then only I can assign you 
Also chose one between this or #395

**Comment by justdvnsh** [comment]
@vj-codes Yeah. I contribute regularly to mentorship-android however, this is my first time in this repo. 
And firstly I'd like to take up this issue then #395

**Comment by vj-codes** [comment]
Assigning you @justdvnsh 
Happy coding!

**Comment by justdvnsh** [comment]
@vj-codes Thanks !

**Comment by justdvnsh** [comment]
Hi. @vj-codes @rpattath . I think this is not an issue. i do not see anywhere where status code of 201 is being returned. Status code of ```HTTPStatus.OK``` is being returned which is status code 200. Even in the tests the checks are fine.

![Screenshot from 2020-10-17 04-54-14](https://user-images.githubusercontent.com/29144058/96322019-1f566980-1035-11eb-9136-53e6bc80671c.png)

![Screenshot from 2020-10-17 04-56-59](https://user-images.githubusercontent.com/29144058/96322049-35642a00-1035-11eb-87ba-b0898a05d438.png)

**Comment by mtreacy002** [comment]
@justdvnsh , @vj-codes and @rpattath . I think this issue should've been closed when I opened the issue #623 which got fixed by PR #637. I must've overlooked this issue since at the time issue #623 was opened, MS was refactoring all hard-coded HTTP error code to HTTPStatus, hence, this issue was no longer valid (as it is using hard-coded error code). I'll close this issue now. Thank you @justdvnsh for pointing this out.
